packages/ui/react-ui-stack/src/components/Section.tsx

```diff
@@ -18,120 +18,148 @@ import {
+                    'm-1 shrink-0',
+                    hoverableControlItem,
+                    hoverableFocusedControls,
+                    hoverableOpenControlItem,
+                    active === 'overlay' && 'invisible',
+                  ]}
+                  data-testid='section.options-menu'
+                >
+                  <DotsThreeVertical className={getSize(4)} />
+                </Button>
+              </DropdownMenu.Trigger>
+
+              <DropdownMenu.Portal>
+                <DropdownMenu.Content>
+                  <DropdownMenu.Viewport>
+                    <DropdownMenu.Item onClick={onNavigate} data-testid='section.navigate-to'>
+                      <ArrowSquareOut className={mx(getSize(5), 'mr-2')} />
+                      <span className='grow'>{t('navigate to section label')}</span>
+                    </DropdownMenu.Item>
+                    <DropdownMenu.Item onClick={onRemove} data-testid='section.remove'>
+                      <X className={mx(getSize(5), 'mr-2')} />
+                      <span className='grow'>{t('remove section label')}</span>
+                    </DropdownMenu.Item>
+                  </DropdownMenu.Viewport>
+                  <DropdownMenu.Arrow />
+                </DropdownMenu.Content>
+              </DropdownMenu.Portal>
+            </DropdownMenu.Root>
           </div>
-        </ListItem.Root>
-      </DensityProvider>
+        </div>
+      </ListItem.Root>
+    </DensityProvider>
+  );
+});
+
+const SECTION_TILE_NAME = 'SectionTile';
+
+const SectionTile: MosaicTileComponent<StackSectionItemWithContext, HTMLLIElement> = forwardRef(
+  ({ path, type, active, draggableStyle, draggableProps, item }, forwardedRef) => {
+    const { t } = useTranslation(translationKey);
+    const { activeItem } = useMosaic();
+
+    const { transform, onRemoveSection, onNavigateToSection, SectionContent, ...contentItem } = item;
+
+    if (!SectionContent) {
+      console.warn('[todo]', 'falsy SectionContent');
+    }
+
+    const transformedItem = transform
+      ? transform(
+          contentItem,
+          // TODO(wittjosiah): `active` doesn't always seem to be accurate here.
+          activeItem?.item.id === contentItem.id ? activeItem?.type : type,
+        )
+      : contentItem;
+
+    const section = (
+      <Section
+        ref={forwardedRef}
+        id={transformedItem.id}
+        title={transformedItem.object?.title ?? t('untitled section title')}
+        active={active}
+        draggableProps={draggableProps}
+        draggableStyle={draggableStyle}
+        onRemove={() => onRemoveSection?.(path)}
+        onNavigate={() => onNavigateToSection?.(transformedItem.object.id)}
+      >
+        {SectionContent && <SectionContent data={transformedItem.object} />}
+      </Section>
     );
+
+    return active === 'overlay' ? <List>{section}</List> : section;
   },
 );
+
+SectionTile.displayName = SECTION_TILE_NAME;
+
+export { Section, SectionTile };
```

Comment:
is there a reason for this vs `export const`? generally I'd favour consistency within the codebase

Reply:
This is how all exports are done in `react-ui` core, which is to align with Radix exports which look like this. I believe it’s so the call to set `.displayName` looks sure.

Reply:
Essentially I use this export style in any module that sets `.displayName`, which is done to improve ergonomics if anything gets logged about `useContext`.

Using a StackContext didn’t help with this solution, though, so I’ll see about reverting.

Reply:
Was able to revert 👍